src/Task2Test.kt

```diff
@@ -0,0 +1,37 @@
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import test.agileFreaks.firstTwoLargestIndexes
+import kotlin.test.*
+
+
+internal class Task2Test {
+
+    @Test
+    //Exceptional case where index -1 is returned
+    // to show that an array of length of less than
+    // 2 was given as input.
+    fun onEmptyArray() {
```

Comment:
Same problem as for problem 1.

The whole point of the TDD is the play between tests and code, you write a little bit of code and then some code to make the test pass and then refactor if the case might be and then start over.